### ISSUE TITLE
Rewarding national selections

### ISSUE THREAD

**Issue by LVoronina** [vbonvin/IPT-public#10]
One potential issue that I foresee related to the proposed system is that there might be a case when an IOC member announced that he/she would hold a national tournament, used the problems chosen for the pre-selection in the promotion, but ended up with one team.  This might be the case in Switzerland, for example.  Then this one team has an unfair advantage compared to other countries, where there is also one team, but the problems for the pre-selection are not known in advance.  

I don’t yet have a clear idea how to overcome this.  In the end, maybe it is not a big deal. I believe that the fact of having the national competition already gives a huge advantage by itself.

**Comment by vbonvin** [comment]
Straight from Issue #9 : 

> We need also to be careful when communicating about it. What if some countries ask in advance to know which problems will be in the pre-selection, but do not organize a national selection in the end ? Do we minor their pre-selection grade somehow, e.g. multiplying the final grade by 0.8 ? Or do we count on their good faith ?

**Comment by vbonvin** [comment]
I personally think that we should try to be as complete as possible in the rules, and try to plan for every possible outcomes even if we currently have no reason to doubt the good faith of the IOC members. 

We also need to state what do we think a minimal national selection is. To me, that would be at least two teams of four people, competing on at least three problems from the official list.

**Comment by LVoronina** [comment]
I agree with such definition of the NS.  

And fully agree that the rules should assume any possible outcome.  But how can we ensure that only the “real” NSs get the advantage?  

If we communicate the problems of the pre-selection to everybody, then all the teams, who do not take part in the NS get a large advantage, since they can concentrate on one problem out of 3 in the pre-selection, while for the NS you have to solve at least 2 of them.  So I think,  publishing the problems for the pre-selection early and visible for all is not an option.

I see only one way: we give out the problems only when there is 2 teams registered for the NS.  If the NS does not happen because one of teams does not show up, we let it pass once and see if we get a feeling that this situation is misused.  I don’t think that it is fair to punish a team by deducing their points, if the team of another Uni did not show up.

Other ideas?

**Comment by vbonvin** [comment]
We could say that we give the problems if at least two teams are registered and that the national selection gets a proper exposure on the related national IPT website,  i.e. we can see the names of the participants, where the national selection will be held, and so on...  Then indeed, if one team does not show up it wouldn't be nice to punish the other team, so we let it pass.

**Comment by LVoronina** [comment]
I would agree with that.  We will need to state all what you (V) said (and possibly other criteria) formally in the rules, when we describe what is a NS.

**Comment by vbonvin** [comment]
It's true that if the national selection informations are put somewhere on a website, the list of problems is not private anymore and the advantage is lost.

What about doing it this way: we select three problems for the preselection. The first problem can be submitted   for the preselection only by teams that took part in a national selection. The second and third problems are announced when the registration closes and can be submitted for the preselection by any team.

Thus, the countries with a national selection give their team more time to work on a problem that can be submitted in case of preselection. Yet, if they prefer to submit another problem, they also have that choice but they don't get more time than the other teams.

**Comment by LVoronina** [comment]
I sounds sophisticated, but I think I agree.  An the question of what is a true NS becomes less critical as well, as we will judge if the NS happened AFTER it happened and not before.

So do I get it right that we first choose 1 problem for the pre-selection, inform everybody about it, but we say that only the team with NS can submit solutions for this problem.  And then we choose and announce another 2 problems for the pre-selection, which everybody can solve.

**Comment by vbonvin** [comment]
Yes Lucy, that's exactly that !

**Comment by Vovcick** [comment]
Vivien and Lucy, as for me, the proposed procedure is rather complicated... And the main bonus for that team with NS will be not more time for preparison but just the increased number of possible problems to prepare (3 instead of 2) which is mainly probabilistic advantage (the probability of having such a problem that a team is able to prepare well increases).
One more small issue could be that it often happens when you want to organize a NS in certain country that you have planned it to take place and had 2 or 3 teams willing to play but only one team remained until the end. And if they would pay too much focus on a certain problem than it could reduce their successes on the whole list of other problems.
One more consideration could be about the publication of two remaining problems just after the registration closes e.g. after some long time. It could lead to a strong disadvantage for some teams with NSs if their NS problems lists didn't include these problems. The other teams worked on all the problems all this time but the teams from this country have priority to these 10 or 12 problems that will be played on their NS and they will have very small time to prepare one of these two problems in case that the first problem that was announced by us is unacceptable for them for some reason.
In any case you may consider this like a cryticism but I'm not fully against the proposed procedure,  Maybe we may just consider some another variants of rewarding like in case that such a team has marks very close to a team without the NS in the preselection we are giving the priority to the first one.

**Comment by vbonvin** [comment]
I think you understood the idea wrong. Let me clarify:

> And the main bonus for that team with NS will be not more time for preparison but just the increased number of possible problems to prepare (3 instead of 2)

No, the main bonus is that the team with NS can submit a problem they have been working of for their national selection! We announce the first problem before the NS take place, so each country with a NS can include it in their local problem list.

> One more small issue could be that it often happens when you want to organize a NS in certain country that you have planned it to take place and had 2 or 3 teams willing to play but only one team remained until the end.

In that case, the team is not allowed to submit a preselection report on the problem reserved to the countries with a national selection.

> One more consideration could be about the publication of two remaining problems just after the registration closes e.g. after some long time. It could lead to a strong disadvantage for some teams with NSs if their NS problems lists didn't include these problems.

No, all the teams discover these two remaining problems at the same time so there is no disadvantage. The only thing is that the teams with national selections have, in addition to these two problems, the ability to submit a preselection report on a problem they have already been working on earlier.

> In any case you may consider this like a cryticism but I'm not fully against the proposed procedure. Maybe we may just consider some another variants of rewarding like in case that such a team has marks very close to a team without the NS in the preselection we are giving the priority to the first one.

I'm obviously open to any other suggestions. But in what you suggest, how do you define "very close" without being completely arbitrary ?

A simpler thing we can do is let each team submit a report on the problem of their choosing. But then it makes it more complicated to be fair in the grading.

**Comment by LVoronina** [comment]
Talking about simpler option, suggested by Vivien, why would it be more complicated to be fair?  I mean, yes, it would be more complicated for us because we would have to inform ourselves about more problems, but otherwise, it’s the same as we do in the final: each team presents the problem they want.

**Comment by vbonvin** [comment]
I find it easier to be fair when the reports/presentations are good. But the important part of the preselection is to decide who are the worst. There is a limited number of way to solve properly a problem, but an infinite amount of way to screw up.

But if you both prefer to let the team present a problem of their choosing then we will proceed like this. It still gives a slight advantage to the teams that have a national selection, which is the most important point for me here.

**Comment by LVoronina** [comment]
By now I start to have a feeling that it’s the most srightforward thing to do.

What Vladimir was saying previously, makes sense to me.  The way I understood it, is that it’s not guarantied that the teams from the NSs will want to solve that very problem that we suggest them to.  And if not, there only real advantage is the ability to choose out of 3 problems and not of 2. 

On the other hand, when we have to choose other 2 problems, it’s difficult not to be biased towards the problems that already were chosen for certain NS, swiss or french, for example, given the current execom.

**Comment by vbonvin** [comment]
> On the other hand, when we have to choose other 2 problems, it’s difficult not to be biased towards the problems that already were chosen for certain NS, swiss or french, for example, given the current execom.

Hence the interest of choosing them early, before the countries pick them for their NS.

**Comment by Vovcick** [comment]
1. Ok, Vivien, maybe I was also not clear enough. Yes, the NS team will have more time for preparation on this single problem, but if we consider them a weak team - than a probability that this problem will be among the topics they are good in is low. And when we add two more problems this probability may increase in 2-3 times even if they have less time for their solution. I wanted to say that this thing is more limiting factor than time for problem solution for rather weak teams and maybe because of that the main advantage will be just increased probability of a suitable problem existence in the list :-) Surely my model is a little bit exaggerated but Lucy at least finds some sense in it)
2. Yes but they may have spent a lot of time on it and their good intentions may had been broken by some unpredictable factors) Like two other teams didn't keep their promises and couldn't come - a frequent situation.
3. I mean the situation when the registration ends let's say in November and the NS will take place in January. And the preselection problems should be sent a week after their NS. Surely, the teams which struggle to win NS will not focus on preselection problems and will pay main attention to the NS. That means that if these two problems were not included into the initial NS list nobody will try to prepare them before they will win a NS and if these problems will be good enough than a team will have only a week to prepare one of them and it's a disadvantage in this case for a country with NS :-)

About letting the teams with NS to choose their best problem it's too big advantage I think. As it often happens when a team has solved one problem very well (for example when a student is just working in this field) and another problems are solved very poor. Also we may choose three problems that are not the most interesting and there will be another problems which will be surely the leaders because of their originality and overall condition pleasantness even if not solved at the corresponding quality. Iit is a big complication to evaluate such different problems for us and we would have to find specialists in any of these fields or to become them by ourselves :-)

**Comment by LVoronina** [comment]
Then we will tend to implement them in our NSs :)

**Comment by vbonvin** [comment]
I'm not really convinced by your arguments Vladimir, but we need to converge towards something. What about the following:

We select, say, four problems that we announce as soon as possible. We choose two for which only the teams that comes from a NS can submit a preselection report on. The two others are open to everyone.

> About letting the teams with NS to choose their best problem it's too big advantage I think. As it often happens when a team has solved one problem very well (for example when a student is just working in this field) and another problems are solved very poor. 

As the ranking of the preselection is not that important, I think such a doing is not that bad. We let the teams chose the problem they prefer, and we see what they are capable of.

**Comment by LVoronina** [comment]
If we announce 4 problems for the pre-selection, all of them most probably will be included in the NSs.  Then the advantage that the teams from the NSs will get is only a broader choice of problems (not 2 but 4).  I think, it is quite a vague advantage.  

I propose to simplify the story.  We choose 3 problems for the pre-selection, announce them ASAP to all IOC and that’s it.  These 3 problems will be included in the NSs.  The advantage of the teams from the NSs will be that they has a stimulus to work on these problems in advance, they play them, discuss them and like that improve their pre-selection report.  This will almost guarantee that the teams from the NSs will pass.   

Now, other teams, who are so motivated that they will start to work on a problem for the pre-selection way in advance, are worth accepting anyway.  

In the end, our goal is to ensure that the teams from the NSs will pass.  The scheme above does it.

What do you think?

**Comment by vbonvin** [comment]
> If we announce 4 problems for the pre-selection, all of them most probably will be included in the NSs.

That's the idea! Not only will they have a broader choice of problems for the preselection but they will be able to work and debate on it with other teams from their country. What you suggest simply removes the "4 choices over 2" from my proposition and in that case.

> This will almost guarantee that the teams from the NSs will pass.

That's not exactly what I want to achieve with this: what I want is that it gets really worth organizing a NS over just sending one single team. With your suggestion, it will be less effort for a team to work from the beginning on one of the problems from the preselection than having to compete on more problems for their NS.

In the end I'm pretty sure that all the teams with a NS will pass the preselection anyway, regardless what we choose. Speaking frankly, the real point behind this rule is to make the IOC members believe that organizing a NS is a big advantage.

**Comment by LVoronina** [comment]
> In the end I'm pretty sure that all the teams with a NS will pass the preselection anyway, regardless what we choose. Speaking frankly, the real point behind this rule is to make the IOC members believe that organizing a NS is a big advantage.
> 
> 

Ok, I get your point.  Then let’s do how you are suggesting this year and see what we get.  

Our actions have to be an interplay between promoting the NS ( for which we could even give free spots in the pre-selection), and making sure we don’t give green light to the weak NSs, like it was last year in Iran… I do not have a better idea how to combine both.

**Comment by Vovcick** [comment]
I agree with Vivien, it's more or less good solution if we want to try to promote somehow making the NSs. But I think here is more psychological effect than real as particpating in NS is for sure itself a very big advantage and motivation for team as Lucy said.

We may try to increase this very psychological effect just how our politics increase or decrease the price of oil or smth. by making some allegations :-)
Like we may declare:
"The countries making a serious enough national selection have a big advantage in the preselection process. Statistics shows that no one of such countries have failed the prelimination stage."

**Comment by vbonvin** [comment]
So we do agree on that, good.

Now, we have to chose the four problems. To my knowledge, only France and Ukraine have already chosen the problems for their NS. We don't care about France as they are by default qualified, but I think it might be good to have at least one of "NS-only" problem that is in all the NS that have already chosen their. Do you know if there are other countries than Ukraine that have already selected their problems ?

**Comment by vbonvin** [comment]
I would suggest the following problems:

For NS only: 1 - Handy glider + 5 - Popcorn
For everyone: 3 - Lego Tower + 15 - Tea with honey

These four problems seems quite easy to perform experiments on. What do you think ?
